run-cases.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
```

Comment:
Add a short comment describing what this script does, and add short comments throughout to explain the script, or at least provide the main steps.